flow/scenarios/loop_merge.py

```diff
@@ -63,7 +63,7 @@ def __init__(self,
 
         num_vehicles = vehicles.num_vehicles
         num_merge_vehicles = sum("merge" in vehicles.get_type(veh_id)
-                                 for veh_id in vehicles.get_ids())
+                                 for veh_id in vehicles.ids)
```

Comment:
hmm?

Reply:
this is calling the `VehicleParams` object instead of the vehicles kernel. This is just to create a distinction in usage as well